app/_hub/kong-inc/degraphql/_index.md

```diff
@@ -27,113 +26,127 @@ params:
   name: degraphql
   service_id: true
   dbless_compatible: 'yes'
-  config: null
+  config:
+    - name: graphql_server_path
+      required: false
+      default: "/graphql"
+      datatype: string
+      description: The path to the GraphQL server.
```

Comment:
For reviewers: I don't know if there's much more to say about the server path param. Maybe an example? But even then, there's already one in the "default" field.